code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -2774,31 +3669,133 @@ Follow by example and make good judgement based on length which list to include
 		body_parts = list("chest")
 		species_allowed = list("Unathi")
 
-	spelunker
-		name = "Spelunker"
-		icon_state = "spelunker"
-		body_parts = list("l_leg","r_leg","l_arm","r_arm","chest","groin","head")
-		species_allowed = list("Vaurca")
+	una_paw_socks
+		name = "Socks Coloration (Unathi, All)"
+		icon_state = "una_pawsocks"
+		body_parts = list("l_foot","r_foot","l_leg","r_leg","l_arm","r_arm","l_hand","r_hand")
+		species_allowed = list("Unathi")
+
+		chest
+			name = "Socks Coloration (Unathi, Chest)"
+			body_parts = list("chest")
+
+		groin
+			name = "Socks Coloration (Unathi, Groin)"
+			body_parts = list("groin")
+
+		left_arm
+			name = "Socks Coloration (Unathi, Left Arm)"
+			body_parts = list("l_arm")
+
+		right_arm
+			name = "Socks Coloration (Unathi, Right Arm)"
+			body_parts = list("r_arm")
+
+		left_hand
+			name = "Socks Coloration (Unathi, (Left Hand)"
+			body_parts = list("l_hand")
+
+		right_hand
+			name = "Socks Coloration (Unathi, Right Hand)"
+			body_parts = list("r_hand")
+
+		left_leg
+			name = "Socks Coloration (Unathi, Left Leg)"
+			body_parts = list("l_leg")
+
+		right_leg
+			name = "Socks Coloration (Unathi, Right Leg)"
+			body_parts = list("r_leg")
+
+		left_foot
+			name = "Socks Coloration (Unathi, Left Foot)"
+			body_parts = list("l_foot")
+
+		right_foot
+			name = "Socks Coloration (Unathi, Right Foot)"
+			body_parts = list("r_foot")
+
+//Vaurcae specific
 
 	delver
 		name = "Delver"
 		icon_state = "delver"
 		body_parts = list("l_leg","r_leg","l_arm","r_arm","chest","groin","head")
 		species_allowed = list("Vaurca")
 
+	spelunker
+		name = "Spelunker"
+		icon_state = "spelunker"
+		body_parts = list("l_leg","r_leg","l_arm","r_arm","chest","groin","head")
+		species_allowed = list("Vaurca")
+
+//Skrell specific
+
+	skr_arms
+		name = "Skrell Arms (All)"
+		icon_state = "skrell_arms"
+		body_parts = list("l_arm","r_arm","l_hand","r_hand")
+		species_allowed = list("Skrell")
+
+		left_arm
+			name = "Skrell Arms (Left Arm)"
+			body_parts = list("l_arm")
+
+		right_arm
+			name = "Skrell Arms (Right Arm)"
+			body_parts = list("r_arm")
+
+		left_hand
+			name = "Skrell Arms (Left Hand)"
+			body_parts = list("l_hand")
+
+		right_hand
+			name = "Skrell Arms (Right Hand)"
+			body_parts = list("r_hand")
+
+	skr_circles_left
```

Comment:
If those are the markings that are on the head, won't work and those sprites should be removed, due to the hair covering them.

Reply:
I stuck a fat WIP sign for a reason and this is exactly what I intend to work out. It would be a wasted opportunity just because we're too ceebs to find a way. No-one said it would be easy.

I have ideas to start with because I know other servers have managed to achieve this.

Reply:
I am with you on the augments, though. Headcase notwithstanding, they can still be used to represent minor things, though. Think SWTOR cyborg, or like, not augments.



The 'cyclops one' was never   a   thing...